router/core/graph_server.go

```diff
@@ -834,7 +874,7 @@ func (s *graphServer) buildGraphMux(ctx context.Context,
 			Subgraphs:                configSubgraphs,
 			RouterEngineConfig:       routerEngineConfig,
 			PubSubProviders:          s.pubSubProviders,
-			Reporter:                 s.websocketStats,
+			Reporter:                 s.engineStats,
```

Comment:
How does it behave with multiple mux? Let's meet about this. In theory each mux has different base attributes and different stats.

Reply:
Decided to cover all with the base graph config